includes/controllers/class-attribute.php

```diff
@@ -174,6 +174,12 @@ function( $name ) {
 			return hp\rest_error( 403 );
 		}
 
+		if ( 'listing' === $model_name && 'listing_attributes' === $meta_box ) {
+
+			// Save listing temporary category ID.
+			update_post_meta($post->ID, 'hp_listing_temporary_category', $request->get_param( 'hp_category' ));
```

Comment:
Check if there's another solution, if the meta box rendering occurs within the same REST API request, then storing this in the database is not needed, there are 2 ways:

1. Setting category for the model object which is checked in the Attribute component when it adds meta box fields (I remember we did this somewhere)
2. Using a request context variable via hivepress()->request->set_context()

Reply:
Firstly, it works with a first way but then it was decided to refuse the ability to save a category without clicking on the "save" post button

Reply:
I've tried second way and it works pretty well